.github/workflows/build.yml

```
@@ -62,12 +62,12 @@ jobs:
     needs: ['lint', 'unit-test']
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
-      - name: Set up Go 1.15
+      - name: Set up Go 1.19
         uses: actions/setup-go@v3
```

Maybe this will be useful here?
```suggestion
        uses: actions/setup-go@v4
```